docs/guides/monitor/lamp-stack.md

```diff
@@ -0,0 +1,227 @@
+  application.
+- [PHP](https://www.php.net/) is a scripting language used to query the MySQL database and build new pages.
+
+LAMP stacks are the foundation for tons of end-user applications, with [Wordpress](https://wordpress.org/) being the
+most popular.
+
+## Challenge
+
+You've already deployed a LAMP stack, either in testing or production. You want to monitor every service's performance
+and availability to ensure the best possible experience for your end-users. You might also be particularly interested in
+using a free, open-source monitoring tool.
+
+Depending on your monitoring experience, you may not even know what metrics you're looking for, much less how to build
+dashboards using a query language. You need a robust monitoring experience that has the metrics you need without a ton
+of required setup.
+
+## Solution
+
+In this tutorial, you'll set up robust LAMP stack monitoring with Netdata in just a few minutes. When you're done,
+you'll have one dashboard to monitor every part of your web application, including each essential LAMP stack service.
+
+This dashboard updates every second with new metrics, and pairs those metrics up with preconfigured alarms to keep you
+informed of any errors or odd behavior.
+
+## What you need to get started
+
+To follow this tutorial, you need:
+
+- A physical or virtual Linux system, which we'll call a _node_.
+- A functional LAMP stack. There's plenty of tutorials for installing a LAMP stack, like [this
+  one](https://www.digitalocean.com/community/tutorials/how-to-install-linux-apache-mysql-php-lamp-stack-ubuntu-18-04)
+  from Digital Ocean.
+- Optionally, a [Netdata Cloud](https://app.netdata.cloud/sign-up?cloudRoute=/spaces) account, which you can use to view
+  metrics from multiple nodes in one dashboard, and a whole lot more, for free.
+
+## Install the Netdata Agent
+
+If you don't have the free, open-source [Netdata Agent](/docs/get/README.md) installed on your node yet, get started
+with a [single kickstart command](/packaging/installer/methods/kickstart.md):
+
+```bash
+bash <(curl -Ss https://my-netdata.io/kickstart.sh)
+```
+
+The Netdata Agent is now collecting metrics from your node every second. You don't need to jump into the dashboard yet,
+but if you're curious, open your favorite browser and navigate to `http://localhost:19999` or `http://NODE:19999`,
+replacing `NODE` with the hostname or IP address of your system.
+
+## Enable hardware and Linux system monitoring
+
+There's nothing you need to do to enable [system monitoring](/docs/collect/system-metrics.md) and Linux monitoring with
+the Netdata Agent, which autodetects metrics from CPUs, memory, disks, networking devices, and Linux processes like
+systemd without any configuration. If you're using containers, Netdata automatically collects resource utilization
+metrics from each using the [cgroups data collector](/collectors/cgroups.plugin/README.md).
+
+## Enable Apache monitoring
```

Comment:
We could also add the web server log collector: https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/weblog

The only gotcha is that the logs must be accessible by the `netdata` user.

Reply:
Usually they are readable for `adm` group members, we add netdata to that group [during install](https://github.com/netdata/netdata/blob/74f3dedc2e16ad28d94b32fd01f5d6648a8bf5a7/netdata-installer.sh#L1108-L1124).

> adm: Group adm is used for system monitoring tasks. Members of this group can read many log files in /var/log, and can use xconsole. Historically, /var/log was /usr/adm (and later /var/adm), thus the name of the group.

Reply:
Hey, good catch, @OdysLam! I guess I had included only a measly sentence about web logs, not an entire section. Just added one now.